extension/android/CMakeLists.txt

```diff
@@ -155,6 +155,26 @@ if(EXECUTORCH_BUILD_LLAMA_JNI)
     ${EXECUTORCH_ROOT}/examples/models/llama2/runner
     ${CMAKE_CURRENT_BINARY_DIR}/../../examples/models/llama2/runner
   )
+
+    target_sources(
```

Comment:
We should protect this under a flag

Reply:
aligned with @kirklandsign that he will help make change to this as it is CMake